drivers/ukbus/pci/include/uk/bus/pci.h

```diff
@@ -150,6 +151,16 @@ enum pci_device_state {
 	PCI_DEVICE_STATE_RUNNING
 };
 
+/**
+ * @brief Description of a mapped BAR.
+ */
+struct pci_bar_memory {
+	/**< Start address of memory region */
+	void *start;
+	/**< Size of the memory region */
```

Comment:
I think `*<` is used for putting comments *after* the variable in Doxygen (although I saw a few places where it is used this way in the codebase).